numba/core/compiler.py

```diff
@@ -601,6 +601,25 @@ def define_nopython_lowering_pipeline(state, name='nopython_lowering'):
         pm.finalize()
         return pm
 
+    @staticmethod
+    def define_parfor_gufunc_nopython_lowering_pipeline(state, name='parfor_gufunc_nopython_lowering'):
```

Comment:
```suggestion
    def define_parfor_gufunc_nopython_lowering_pipeline(
        state, name='parfor_gufunc_nopython_lowering'):
```

☝️ To fix the flake8 error 👇 

numba/core/compiler.py:605:81: E501 line too long (103 > 80 characters)